content/docs/create-a-new-react-app.md

```diff
@@ -8,87 +8,87 @@ prev: add-react-to-a-website.html
 next: cdn-links.html
```

Comment:
Title can be translated @fanyspenda 😄  I think
`Membuat Sebuah Aplikasi React Baru` is a good start, WDYT?